core/src/main/java/cz/xtf/core/helm/HelmBinary.java

```diff
@@ -0,0 +1,57 @@
+package cz.xtf.core.helm;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+import cz.xtf.core.config.OpenShiftConfig;
+import cz.xtf.core.openshift.CLIUtils;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class HelmBinary {
+
+    private final String path;
+    private final String helmConfigPath;
+    private final String kubeUsername;
+    private final String kubeToken;
+    private final String namespace;
+
+    public HelmBinary(String path, String kubeUsername, String kubeToken, String namespace) {
+        this.path = path;
+        this.kubeUsername = kubeUsername;
+        this.kubeToken = kubeToken;
+        Path helmConfigFile = Paths.get(path).getParent().resolve(".config");
+        try {
+            helmConfigFile = Files.createDirectories(helmConfigFile);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        this.helmConfigPath = helmConfigFile.toAbsolutePath().toString();
+        this.namespace = namespace;
+    }
+
+    public HelmBinary(String path, String helmConfigPath, String kubeUsername, String kubeToken, String namespace) {
+        this.path = path;
+        this.helmConfigPath = helmConfigPath;
+        this.kubeUsername = kubeUsername;
+        this.kubeToken = kubeToken;
+        this.namespace = namespace;
+    }
+
+    public String execute(String... args) {
```

Comment:
We talked about some fluent api would be nice for most frequent commands. That is out of scope of this MR?